docs/onboarding/06-e2e-sample.md

```diff
@@ -0,0 +1,31 @@
+# Chapter 6: Deploy and run end-to-end sample
+
+The following docs will introduce an end-to-end solution where an Azure Digital Twins instance is connected to other Azure services for management of devices and data.
+
+## Key Points
+
+- Create an end-to-end scenario that shows Azure Digital Twins being driven by live device data.
+- Understand how changes are propagated through the twin graph by processing digital twin notifications.
+
+## Prerequisites
+
+In addition to the Prerequisites defined in the [README](README.md), the following steps are required to complete the learning exercises below:
+
+- [Set up your session](https://docs.microsoft.com/en-us/azure/digital-twins/tutorial-end-to-end#set-up-cloud-shell-session) and register your Azure Subscription with the Azure Digital Twins namespace
+- [Add the Azure IoT Extension](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-use-cli#get-the-extension) for Azure CLI
+- [Create an Azure Digital Twin](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-set-up-instance-cli#create-the-azure-digital-twins-instance) instance
+- Download the sample project [digital-twins-samples](https://docs.microsoft.com/en-us/azure/digital-twins/tutorial-end-to-end#get-required-resources)
+
```

Comment:
> Note: Use the Visual Studio Installer to ensure your installation of Visual Studio 2019 includes the ASP.NET and web development workload/framework. Without this, an Azure Function cannot be published.

Reply:
Good catch. Thank you Dexter! Included note.

Reply:
Correction: 
> [!NOTE] 
> Ensure your installation of Visual Studio 2019 includes the **[Azure Development Workload](https://docs.microsoft.com/en-us/dotnet/azure/configure-visual-studio)**. This workload enables the application to publish the SampleFunctionApp.

Reply:
Thank you for double checking and testing this!! :)